controller_manager/src/controller_manager.cpp

```diff
@@ -150,17 +150,30 @@ ControllerManager::ControllerManager(
   }
 
   std::string robot_description = "";
+  // TODO(Manuel): robot_description parameter is deprecated and should be removed.
   get_parameter("robot_description", robot_description);
   if (robot_description.empty())
   {
-    throw std::runtime_error("Unable to initialize resource manager, no robot description found.");
+    // set QoS to transient local to get messages that have already been published
+    // (if robot state publisher starts before controller manager)
+    RCLCPP_INFO(get_logger(), "Subscribing to robot_state_publisher for robot description file.");
```

Comment:
```suggestion
    RCLCPP_INFO(get_logger(), "Waiting for ~/robot_description topic...");
```

Reply:
Only included topic name since we do not wait here...